app/controllers/subscriptions_controller.rb

```diff
@@ -5,19 +5,21 @@ class SubscriptionsController < ApplicationController
 
   def create
     @subscription = current_user.subscriptions.build(conference_id: @conference.id)
-    if @subscription.save!
+    if @subscription.save
       redirect_to root_path, notice: "You have been subscribed to receive email notifications for #{@conference.short_title}."
```

Comment:
BTW why don't we also fix this message: You have subscribed.... for @conference.title